### PR TITLE
Fix: add missing thousand separators

### PR DESCRIPTION
**Pull request by oyershov**
Input fields don\'t contain thousands separators to avoid calculation problems.
Снимок экрана 2020-09-01 в 12 26 37

### CODE REVIEW

**Review by akhlopiachyi — APPROVED** [review]

File: src/containers/RecentTrades/Yours.tsx [hunk 1]
```diff
@@ -108,8 +108,12 @@ class RecentTradesYoursContainer extends React.Component<Props> {
 
         return [
             <span style={{ color: setTradesType(orderSide).color }} key={id}>{higlightedDate}</span>,
-            <span style={{ color: setTradesType(orderSide).color }} key={id}><Decimal key={id} fixed={amountFixed}>{amount}</Decimal></span>,
-            <span style={{ color: setTradesType(orderSide).color }} key={id}><Decimal key={id} fixed={priceFixed} prevValue={[...list][i - 1] ? [...list][i - 1].price : 0}>{price}</Decimal></span>,
+            <span style={{ color: setTradesType(orderSide).color }} key={id}>
+                <Decimal key={id} fixed={amountFixed} thousSep=",">{amount}</Decimal>
+            </span>,
+            <span style={{ color: setTradesType(orderSide).color }} key={id}>
+                <Decimal key={id} fixed={priceFixed} thousSep="," prevValue={[...list][i - 1] ? [...list][i - 1].price : 0}>{price}</Decimal>
```
Comment (VladyslavP) [inline src/containers/RecentTrades/Yours.tsx:115]: we copy here `[...list]` twice, should we store it in a variable ?

**Review by VladyslavP — APPROVED** [review]
👍

**Comment by pkucherenk0** [comment]
The market section does not have separators
Selection_999(908)

File: src/containers/RecentTrades/Yours.tsx (hunk 1 above)
Comment (oyershov) [reply src/containers/RecentTrades/Yours.tsx:115]: already refactored in https://github.com/openware/baseapp/pull/300

**Pull request merged**